language-server/modules/langserver-core/src/test/resources/META-INF/services/org.ballerinalang.langserver.commons.eventsync.spi.EventSubscriber

```diff
@@ -0,0 +1 @@
+org.ballerinalang.langserver.eventsync.EventSyncTestSubscriber
```

Comment:
Shall we add a new line here?

Reply:
I was following the other META-INF files and did not keep a new line intentionally. If needed we can add one.